src/components/retrieveData.js

```diff
@@ -0,0 +1,13 @@
+import htmlQuiz from '../data/dummy.json';
+import cssQuiz from '../data/cssQuiz.json';
+import jsQuiz from '../data/jsQuiz.json';
+import { shuffle } from '../helpers/shuffle';
+
+const data = { html: htmlQuiz, css: cssQuiz, js: jsQuiz };
+
+export const retrieveData = quizName => {
+  const quizData = data[quizName];
+  quizData.forEach(question => shuffle(question.answers));
```

Comment:
If the user goes to a url path like `/scss` then this line throws an error because `quizData` is `undefined`. Lets fix this so an error isn't thrown

Reply:
Now function returns empty array in this case, so Quiz component displays that message above.